lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1589,37 +1589,44 @@ namespace Js
         Js::JavascriptError::MapAndThrowError(this, E_FAIL);
     }
 
-    JavascriptFunction* ScriptContext::LoadScript(const wchar_t* script, SRCINFO const * pSrcInfo, CompileScriptException * pse, bool isExpression, bool disableDeferredParse, bool isByteCodeBufferForLibrary, Utf8SourceInfo** ppSourceInfo, const wchar_t *rootDisplayName, bool isLibraryCode, bool disableAsmJs, bool isSourceModule)
+    ParseNode* ScriptContext::ParseScript(Parser* parser,
+        const byte* script,
+        size_t cb,
+        SRCINFO const * pSrcInfo,
+        CompileScriptException * pse,
+        Utf8SourceInfo** ppSourceInfo,
+        const wchar_t *rootDisplayName,
+        LoadScriptFlag loadScriptFlag,
+        uint* sourceIndex)
```

Comment:
Cool, this script flag is being tested by code in module-syntax.js, right? Assume there is no way to have unit tests for the module linking code yet.